src/python/txtai/cloud/storage.py

```diff
@@ -49,11 +49,7 @@ def __init__(self, config):
         self.client = driver(
             config.get("key", os.environ.get("ACCESS_KEY")),
             config.get("secret", os.environ.get("ACCESS_SECRET")),
-            host=config.get("host"),
-            port=config.get("port"),
-            token=config.get("token"),
-            region=config.get("region"),
-        )
+            **{field: config.get(field) for field in ["host", "port", "api_version", "region", "token"] if config.get(field)})
```

Comment:
If we want to get this merged in, for now I'd remove the api_version and secure fields from both here and the workflow file below.